tests/core/daemon/test_daemon.py

```diff
@@ -45,10 +50,10 @@ async def get_b_tools_1(self, get_temp_keyring):
         return await create_block_tools_async(constants=test_constants_modified, keychain=get_temp_keyring)
 
     @pytest_asyncio.fixture(scope="function")
-    async def get_b_tools(self, get_temp_keyring):
+    async def get_b_tools(self, get_temp_keyring):  # xxx
         local_b_tools = await create_block_tools_async(constants=test_constants_modified, keychain=get_temp_keyring)
         new_config = local_b_tools._config
-        new_config["daemon_port"] = 55401
+        new_config["daemon_port"] = 55401  # xxx
```

Comment:
Please take a close look at this line:

`new_config["daemon_port"] = 55401`

Reply:
In my patch (which I'm hoping to land before this) I remove this assignment and make every `create_blockchain_tools` pick an available port and use that.